en/hardware/reference_design.md

```diff
@@ -30,18 +30,13 @@ The diagram below shows the division of bus and functional responsibilities betw
 
 Some Pixhawk-series controllers are built without the I/O board in order to reduce space or complexity, or to better address certain board use-cases.
 
-The I/O board is disabled by setting parameter [SYS_USE_IO=0](../advanced_config/parameter_reference.md#SYS_USE_IO).
-When the I/O board is disabled:
-- The MAIN mixer file is loaded into the FMU (so the "MAIN" outputs listed in the [Airframe Reference](../airframes/airframe_reference.md) appear on the port labeled AUX).
-  The AUX mixer file isn't loaded, so outputs defined in this file are not used.
-- RC input goes directly to the FMU rather than via the IO board.
-
-Flight controllers without an I/O board have `MAIN` ports, but they *do not* have `AUX` ports.
-Consequently they can only be used in [airframes](../airframes/airframe_reference.md) that do not use `AUX` ports, or that only use them for non-essential purposes (e.g. RC passthrough).
-They can be used for most multicopters and *fully* autonomous vehicles (without a safety pilot using RC control), as these typically only use `MAIN` ports for motors/essential controls.
+In this case [SYS_USE_IO](../advanced_config/parameter_reference.md#SYS_USE_IO) is set to `0` so that the I/O driver is not started.
+Because there are no I/O board outputs, the [Actuator Configuration](../config/actuators.md) will not display the MAIN tab.
```

Comment:
It will show the correct tab: e.g. on pixracer it will only show MAIN (no AUX).

The only case where it's not correct, is for boards with the same target, like the Pixhawk 4 vs Pixhawk 4 mini, which does not have an IO. In that case the mini only shows the AUX tab.

Reply:
@bkueng Arrrgggg.  I am now possibly confused. To confirm:
- The bus used for MAIN and AUX is set by the target like `px4_fmu-v5_default` or `px4_fmu-v4_default`
- So for `px4_fmu-v5_default` FMU is set to AUX and IO is set to MAIN .
  - If you run this on Pixhawk 4 then screen-printed labels and actuators tabs are as you would expect.
  - If you run on Pixhawk 4 mini there is no IO so Actuator UI would say AUX and printing would say main.
- For `px4_fmu-v4_default` (Pixracer) there is only and FMU and it is mapped to MAIN - so there is both MAIN tab and main screenwriting.
  - Presumably this works because only Pixracer runs on this target. However you'd have similar effects on Pixhawk mini.

IN summary, the screen printing and displayed bus mostly match.

Reply:
Yes exactly.